### ISSUE TITLE
Wrong version

### ISSUE THREAD

**Issue by RuslanShah** [lbrndnr/nuage-macos#24]
Wrong version. My version is 0.0.4
<img width="287" alt="Screenshot 2023-07-10 at 14 53 31" src="https://github.com/lbrndnr/nuage-macos/assets/129128156/ba0146f6-866a-45f6-bbf7-549e0a944c56">

**Comment by suschizu** [comment]
I can confirm it - version 0.0.4 info.plist still contains short version string 0.0.2  and bundle version 1 (same in download 0.0.3)

```
	<key>CFBundleShortVersionString</key>
	<string>0.0.2</string>
	<key>CFBundleSupportedPlatforms</key>
	<array>
		<string>MacOSX</string>
	</array>
	<key>CFBundleVersion</key>
	<string>1</string>
```

**Comment by lbrndnr** [comment]
Thanks for your feedback! Fixed this in b7a894d :)
